gymnasium/wrappers/lambda_action.py

```diff
@@ -21,7 +21,19 @@
 class LambdaActionV0(
     gym.ActionWrapper[ObsType, WrapperActType, ActType], gym.utils.RecordConstructorArgs
 ):
-    """Applies a function to the ``action`` before passing the modified value to the environment ``step`` function."""
+    """Applies a function to the ``action`` before passing the modified value to the environment ``step`` function.
+
+    Example:
```

Comment:
Is there a way of showing the impact of the LambdaAction wrapper? Currently this doesn't show that the action has been modified

Reply:
We can probably have two environments, one with an unmodified environment and a modified environment and step through the different actions and should the same output as the LambdaAction modifies the action